source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextField.swift

```diff
@@ -3,21 +3,30 @@ import AppKit
 
 class ACRTextField: NSTextField {
     private let config: InputFieldConfig
+    weak var textFieldDelegate: ACRTextFieldDelegate?
     
-    init(frame frameRect: NSRect, config: RenderConfig) {
-        self.config = config.inputFieldConfig
-        super.init(frame: frameRect)
-        initialise()
+    init(dateTimeFieldConfig config: InputFieldConfig) {
+        self.config = config
+        super.init(frame: .zero)
+        initialise(isDateTime: true)
+        setupConstraints()
+    }
+    
+    init(textFieldConfig config: InputFieldConfig) {
+        self.config = config
```

Comment:
```suggestion
    init(config: InputFieldConfig) {
        self.config = config
```
This can just be `config`